fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/panel/panel.xml

```diff
@@ -1,8 +1,14 @@
 <!-- Copyright (c) 2023-2024 FlyByWire Simulations -->
 <!-- SPDX-License-Identifier: GPL-3.0 -->
 
-<!--TODO: Do we even need this files?`If not it should be removed -->
-
 <PlaneHTMLConfig>
-
+    <Instrument>
+        <Name>A380X_FCU</Name>
+		<Electric>
+            <Or>
+                <Simvar name="L:A32NX_ELEC_DC_ESS_BUS_IS_POWERED" unit="Boolean"/>
+                <Simvar name="L:A32NX_ELEC_DC_2_BUS_IS_POWERED" unit="Boolean"/>
+            </Or>
+		</Electric>
+    </Instrument>
```

Comment:
```suggestion
    <Instrument>
        <Name>A380X_FCU</Name>
		<Electric>
            <Or>
                <Simvar name="L:A32NX_ELEC_DC_ESS_BUS_IS_POWERED" unit="Boolean"/>
                <Simvar name="L:A32NX_ELEC_DC_2_BUS_IS_POWERED" unit="Boolean"/>
            </Or>
		</Electric>
    </Instrument>
```

Not sure if this will have any effect on any potential future plans for panel.xml and liveries but take note.

Reply:
(May be better to move this out of here if so)

Reply:
No affect. We cannot have anybody overriding panel.xml at all.